modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -38,11 +38,12 @@ def submit # rubocop:disable Metrics/MethodLength
 
           track_pact_counter auto_claim
           pdf_data = get_pdf_data
-          pdf_mapper_service(form_attributes, pdf_data, target_veteran).map_claim
+          file_number = get_file_number.nil? ? target_veteran.ssn : get_file_number
```

Comment:
couple things here. this calls BGS twice in it's current state. Also, I don't think it's safe to fallback to the SSN here. there's discussion to be had but we're assuming this would only work for a portion of veterans, granted a large portion, but not all. I would say if we don't get a file number from BGS we just terminate the request and raise an error.

Reply:
https://oddball.slack.com/archives/C019XKGS1CN/p1694190685121019 Kayla confirmed that the birls file number is the same as the VBMS file number, so I will remove the call to BGS